snippets/rtt-tracing/rtt-tracing.conf

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2024 Analog Devices, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_RTT_CONSOLE=y
+CONFIG_UART_CONSOLE=n
+
+CONFIG_DEBUG_OPTIMIZATIONS=n
+CONFIG_DEBUG_THREAD_INFO=y
+
+CONFIG_TRACING=y
+CONFIG_SEGGER_SYSTEMVIEW=y
+CONFIG_SEGGER_SYSTEMVIEW_BOOT_ENABLE=n
+CONFIG_SEGGER_SYSVIEW_POST_MORTEM_MODE=n
+
+CONFIG_IDLE_STACK_SIZE=2048
```

Comment:
Is this size portable?

Reply:
Not really, no. This was an artifact of extracting `samples/subsys/tracing/prj_sysview.conf` into a snippet, but it seems to not be necessary. Removed.